### PR TITLE
Issue #104 : MarkAsPaidController/Logging

### PR DESCRIPTION
**Pull request by vendra131**
Issue #104 : MarkAsPaidController/Logging

### CODE REVIEW

File: src/test/java/com/kodekonveyor/completion/MarkAsPaidControllerLoggingTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+    @Test
+    @DisplayName("Error  is logged when the status of work request is not verified.")
+    public void test3() {
+        ThrowableTester.assertThrows(() -> markAsPaidController.call(workRequestEntityData.getId()));
+
+        Mockito.verify(loggerService)
+                .warn(eq(EXPECTED_LOG_API_FAILURE), eq(EXPECTED_FAILURE_STATUS), errorCaptor.capture());
+
+        Assert.assertEquals(errorCaptor.getValue(), CompletionConstantsTestData.EXPECTED_MARK_PAID_ERROR_INVALID_STATUS);
```
Comment (magwas) [inline src/test/java/com/kodekonveyor/completion/MarkAsPaidControllerLoggingTest.java:71]: two assertions in one test case. This should be separate test case.

Comment (vendra131) [reply src/test/java/com/kodekonveyor/completion/MarkAsPaidControllerLoggingTest.java]: requested changes. Captor doesn't seem to be required.

File: src/test/java/com/kodekonveyor/work_request/WorkRequestEntityRepositoryStubs.java [hunk 2]
```diff
@@ -70,4 +71,14 @@ public static void agreedWorkRequest(
...
   }
 
+    public static void verifiedWorkRequest(
+            final WorkRequestRepository workRequestRepository
+    ) {
+        doReturn(Optional.of(WorkRequestEntityTestData.getStatusVerified()))
+                .when(workRequestRepository)
+                .findById(
+                        WorkRequestEntityTestData.VERIFIED_WORK_REQUEST_ID
+                );
```
Comment (szirbucz) [inline src/test/java/com/kodekonveyor/work_request/WorkRequestEntityRepositoryStubs.java:81]: Please take a look at how the exception was tested and use the same stub if it's possible.

Comment (vendra131) [reply src/test/java/com/kodekonveyor/work_request/WorkRequestEntityRepositoryStubs.java]: Made requested changes to use existing stubs.

**Comment by szirbucz** [comment]
testcases=3

**Pull request merged**